Application.hs

```diff
@@ -162,22 +165,18 @@ develMain = develMainHelper getApplicationDev
 -- | The @main@ function for an executable running this site.
 appMain :: IO ()
 appMain = do
-    -- Get the settings from all relevant sources
-    settings <- withEnv $ loadAppSettingsArgs
-        -- fall back to compile-time values, set to [] to require values at runtime
-        [configSettingsYmlValue]
-
-        -- allow environment variables to override
-        useEnv
-
-    -- Generate the foundation from the settings
+    settings <- getAppSettings
     foundation <- makeFoundation settings
-
-    -- Generate a WAI Application from the foundation
     app <- makeApplication foundation
 
-    -- Run the application with Warp
-    runSettings (warpSettings foundation) app
+    command <- parseCommand
+
+    case command of
+        RunServer -> runSettings (warpSettings foundation) app
+        DeleteStaleComments -> handler $ do
```

Comment:
I'm guessing there's a better place to organize these. I wondered if it would be better to make the delete commands task an actual handler module, and then import/use it here.